api/src/main/java/org/pmiops/workbench/monitoring/views/EventMetric.java

```diff
@@ -0,0 +1,58 @@
+package org.pmiops.workbench.monitoring.views;
+
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.tags.TagKey;
+import java.util.Collections;
+import java.util.List;
+
+/** Metric enum values for events to be counted. */
+public enum EventMetric implements OpenCensusView {
+  NOTEBOOK_CLONE("notebook_clone", "Clone (duplicate) a notebook"),
+  NOTEBOOK_DELETE("notebook_delete", "Delete a notebook"),
+  NOTEBOOK_SAVE("notebook_save", "Save (or create) a notebook");
+
+  private final String name;
+  private final String description;
+  private List<TagKey> columns;
+
+  EventMetric(String name, String description, List<TagKey> columns) {
+    this.name = name;
+    this.description = description;
+    this.columns = columns;
+  }
+
+  EventMetric(String name, String description) {
+    this(name, description, Collections.emptyList());
+  }
+
+  @Override
+  public String getName() {
+    return name;
+  }
+
+  @Override
+  public String getDescription() {
+    return description;
+  }
+
+  @Override
+  public String getUnit() {
+    return OpenCensusView.UNITLESS_UNIT;
+  }
+
+  @Override
+  public Class getMeasureClass() {
+    return MeasureLong.class;
+  }
+
+  @Override
+  public Aggregation getAggregation() {
+    return Aggregation.Count.create();
+  }
+
+  @Override
+  public List<TagKey> getColumns() {
```

Comment:
I'm still trying to understand why I need these `TagKey`s, and if they have anything to do with attachments.

Reply:
These are used for [filter and group metrics](https://opencensus.io/tag/key/#2), of which we currently have none. They're for aggregation rather than attachments (and don't match attachment keys).

For this interface, I wanted to span all the inputs for the `View` create method. I'm thinking a bit ahead about pulling this all out into stand-alone library for doing Spring with OpenCensus, and somebody might need it.

I'd like to leave it here for now since it costs us nothing and allows us to use group metrics more easily when we're ready. I could try to make a demo metric using a group if that helps.